testsuite/tests/basic-more/opaque_prim.ml

```diff
@@ -0,0 +1,8 @@
+
+let f x = opaque x
+
+let () =
+  assert(f f == f);
+  assert(opaque 1 = 1);
+  assert(opaque 1. = 1.)
+
```

Comment:
Not really, you would have to add explicitly some code to break it. It is quite difficult to do something like that without noticing...